src/test/java/jalau/at18/Azul/BagTest.java

```diff
@@ -0,0 +1,39 @@
+package jalau.at18.azul;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+
+public class BagTest {
+
+    @Test
+    public void shouldVerifyThereIsBag() {
```

Comment:
name of the test does not match with the logic being tested, which is "fill bag with colors", not "that there is a bag"

Reply:
Changing the test name to shouldVerifyThereIsMixedBag